scalding-core/src/main/scala/com/twitter/scalding/typed/cascading_backend/AsyncFlowDefRunner.scala

```diff
@@ -95,19 +101,37 @@ class AsyncFlowDefRunner extends Writer { self =>
         case None => this
       }
 
-    def addPipe[T](c: Config,
+    /**
+     * Returns true if we actually add
```

Comment:
could you expand on this comment

Returns true if we will add to our forced pipes, false if cache hit and noop. I think?

Reply:
yes. The thing is, writes are side-effects. We only want to take that side effect once. So, if you return true, we should actually do the side effect. I'll update the comment.